.env.gitpod

```diff
@@ -27,7 +27,7 @@ MAXMIND_USER_ID = 123456
 MAXMIND_LICENCE_KEY = "xxxxxxxxxxxxxxxx"
 
 MICROSOFT_PROVIDER_ID = 1234
-MICROSOFT_LOCAL_FILE_DIRECTORY = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
+MICROSOFT_REMOTE_API_ENDPOINT = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
```

Comment:
Would you mind if we kept this? I'll try laying out my reasoning, which was largely `{company}-{provider}-{content}-{date}`, while trying to avoiding having really long variable names:

- microsoft is the _company_. `ms` is a common abbreviation, just like Amazon might is sometimes shortened to `AMZN`, or google `GOOG`. I accept that using `microsoft` might be clearer if you're coming new to it.
- `azure` is the brand Microsoft uses as a provider of hosted cloud services. Just like `aws` is Amazon's brand Amazon Web Services, and `gcp` is Google Cloud Platform the brand from Google/Alphabet. This is why initially the environment variables for the AWS importer used AWS and not Amazon. One is a service from the org, even if Amazon is the _reporting entity_ that publishes sustainability reports.
- we only have `ip-ranges` in use here, not AS networks or any other formulation. So, there is only one way of describing networks inside the data we get from the endpoint. You could arguably use `network-ranges`, as a convention for all bulk import
- the 2022-04-25 format is an ISO format ([ISO 8601]()https://www.iso.org/iso-8601-date-and-time-format.html), specifically designed to remove ambiguity, and be easy to parse. When you remove the spaces, it's much harder to tell if you're following a YYYY-MM-DD format. When you lose the spaces, and just have a string of numbers it becomes ambiguous.  When you don't have was a figures like 20220425. This could be interpreted as _20:22 on the 4th of April_, or if we were working with an earlier year of data like 2012, 20120425 could easily be interpreted as 20th December (i.e. month 12) at 04:25.